src/executables/util_service.c

```diff
@@ -233,12 +235,14 @@ static UTIL_SERVICE_OPTION_MAP_T us_Command_map[] = {
   {ACCESS_CONTROL, COMMAND_TYPE_ACL, MASK_SERVER | MASK_BROKER | MASK_GATEWAY},
   {RESET, COMMAND_TYPE_RESET, MASK_BROKER | MASK_GATEWAY},
   {INFO, COMMAND_TYPE_INFO, MASK_BROKER | MASK_GATEWAY},
-  {SC_COPYLOGDB, COMMAND_TYPE_COPYLOGDB, MASK_HEARTBEAT},
-  {SC_APPLYLOGDB, COMMAND_TYPE_APPLYLOGDB, MASK_HEARTBEAT},
   {GET_SHARID, COMMAND_TYPE_GETID, MASK_BROKER},
   {TEST, COMMAND_TYPE_TEST, MASK_BROKER},
+#if defined (ENABLE_UNUSED_FUNCTION)
+  {SC_COPYLOGDB, COMMAND_TYPE_COPYLOGDB, MASK_HEARTBEAT},
```

Comment:
Nit. COMMAND_TYPE_COPYLOGDB can be removed along. The same goes for the others.

Reply:
Agreed.
Other parts related to copylogdb/applylogdb will be guarded with `#if 0` in another issue (to minimize changes for now).